tests/phpunit/tests/blocks/register.php

```diff
@@ -252,6 +252,22 @@ public function test_success_register_block_script_handle() {
 		$this->assertSame( 'unit-tests-test-block-script', $result );
 	}
 
+	/**
+	 * @ticket 55513
+	 */
+	public function test_success_register_block_script_handle_in_theme() {
+		switch_theme( 'block-theme' );
+
+		$metadata = array(
+			'file'   => DIR_TESTDATA . '/themedir1/block-theme/blocks/example-block/block.json',
+			'name'   => 'block-theme/example-block',
+			'script' => 'file:./index.js',
+		);
+		$result   = register_block_script_handle( $metadata, 'script' );
```

Comment:
So this one tres to register the `script` while the theme reigsters the `editorScript`, right?